optuna/artifacts/_gcs.py

```diff
@@ -11,7 +11,7 @@
 if TYPE_CHECKING:
     from typing import BinaryIO
 
-with try_import():
+with try_import(package_name="google-cloud-storage") as _imports:
```

Comment:
[google](https://pypi.org/project/google) and [google-cloud-storage](https://pypi.org/project/google-cloud-storage) are completely different packages, so I would like to add `package_name` to avoid confusion.